src/js/newsletter-subscription.js

```diff
@@ -1,5 +1,27 @@
 const NEWSLETTER_SUBSCRIBE_URL = 'https://www.mozilla.org/en-US/newsletter/';
 
+/** Opinionated function for extracting data from the form without needing FormData
+ *
+ * @param {object} form element
+ * @returns {string} URI-encoded string
+ */
+function getPayloadString(form) {
+  try {
+    return new URLSearchParams(new FormData(form)).toString();
+  } catch (err) {
```

Comment:
@Gregoor I think this approach is a bit too 'Pythonic' and it might be better/safer to check if `URLSearchParams` exists, but I can't do that without raising an exception (eg `typeof(URLSearchParams === "undefined")` raises an exception. Any advice?

Reply:
Whoops missed that one. You were close, the parantheses lead to a wrong order of execution. It should be: `typeof URLSearchParams === "undefined"`.

Otherwise you get the following evaluation in IE11:
`typeof(URLSearchParams === "undefined")`
`typeof(throwReferenceError('URLSearchParams') === "undefined")`

and in browsers where it exists you'd get
`typeof(URLSearchParams === "undefined")`
`typeof(false)`
`"boolean"`

Reply:
Ugh, that parenthesis mistake was me fat-fingering things in the IE11 console via Browserstack. Thanks for paying it out so clearly